x-pack/plugins/canvas/kibana.json

```diff
@@ -5,6 +5,6 @@
     "configPath": ["xpack", "canvas"],
     "server": true,
     "ui": false,
-    "requiredPlugins": []
+    "requiredPlugins": [],
+    "optionalPlugins": ["usageCollection"]
```

Comment:
Is there a reason for making it optional vs required?

Reply:
I could be wrong about this reason but I suspect that users (like government folks in airgap environments) don't want that bundled. 

In general though, I was following the conventions set here: https://github.com/elastic/kibana/blob/master/src/plugins/usage_collection/README.md